resources/translations/fr_FR/machine_settings/machine_always_write_active_tool.md

```diff
@@ -1,19 +1,19 @@
-Toujours écrire Outil actif
+Toujours écrire l outil actif
```

Comment:
Shouldn't that have an apostrophe then?

```suggestion
Toujours écrire l'outil actif
```

Reply:
Keeping in mind that my French is high school level + some extra self-learning out of interest. You're the expert here :) But this might be a typo.

Reply:
Yes it should be. But with the apostrophe it was returning an error message. And as ut is not a big issue i have decided  to remove it

Reply:
I'll have to look into what is causing that test to fail with a false negative then. It's not something that the automated test should fail on! Thanks for the correction.